src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TEMPVD

```diff
@@ -1,5 +1,5 @@
-{"name" : "TEMPVD", "sections" : ["PROPS"],
-"items" : [
-  {"name" : "DEPTH", "value_type" : "DOUBLE", "dimension" : "Length"},
-  {"name" : "TEMP", "value_type" : "DOUBLE", "dimension" : "Temperature"}]}
-
```

Comment:
Hmm - the changes to the `TEMPVD`keyword are they intentional?

Reply:
I will fix this.